build.gradle

```diff
@@ -142,6 +142,18 @@ immaculate {
             return sb.toString()
         }
 
+        // Disallow explicit not null in patches, it's always implied.
+        custom 'noNotNull', { String fileContents ->
+            fileContents.eachLine {
+                if (!it.startsWith("+")) return
+                if (it.contains('@NotNull') || it.contains('@Nonnull')
+                        || it.contains('@org.jetbrains.annotations.NotNull')
+                        || it.contains('@javax.annotation.Nullable')) {
```

Comment:
Is there any particular reason to include javax `Nullable` in the non-null handling? Seems confusing and also redudandant since it's also handled a right below this.

Reply:
Oh that's a typo...